datadog-cloudformation-common-python/src/datadog_cloudformation_common/utils.py

```diff
@@ -0,0 +1,19 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache 2.0 style license (see LICENSE).
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2020-Present Datadog, Inc.
+from cloudformation_cli_python_lib import HandlerErrorCode
+
+
+def http_to_handler_error_code(http_error_code: int) -> HandlerErrorCode:
+    if http_error_code == 400:
+        return HandlerErrorCode.InvalidRequest
+    elif http_error_code == 403:
+        return HandlerErrorCode.AccessDenied
+    elif http_error_code == 404:
+        return HandlerErrorCode.NotFound
+    elif http_error_code == 409:
+        return HandlerErrorCode.ResourceConflict
+    elif http_error_code == 422:
+        return HandlerErrorCode.ServiceLimitExceeded
```

Comment:
Hmm not sure if 422 would translate to ServiceLimitExceeded here. ServiceLimitExceeded seems to suggest some sort limit was reached on the service side. Maybe we can use `GeneralServiceException` here since there does not seem to be a good alternative. What do you think?

Reply:
Oh yeah, I confused it with 429 for rate limits.

Reply:
Or with the 402 we send back when we reach quota. Probably a mixture of both, good catch haha